compute/instances_test.go

```diff
@@ -30,6 +35,202 @@ func getAnyInstanceID(t *testing.T, client *compute.ComputeClient) (string, erro
 	return "", errors.New("no machines configured")
 }
 
+func RandInt() int {
+	reseed()
+	return rand.New(rand.NewSource(time.Now().UnixNano())).Int()
```

Comment:
I'm going to stump the seeding library here, either for explicit use or to crib an example from: https://github.com/sean-/seed